src/jqLite.js

```diff
@@ -439,6 +440,27 @@ forEach({
       return  val;
     }
   },
+  
+  offset: function(element, name, value) {
+    var documentElem,
+      box = { top: 0, left: 0 },
+  		doc = element && element.ownerDocument;
+  
+  	if (!doc || isDefined(name)) { 
+  		return;
+  	}
+  
+  	documentElem = doc.documentElement;
+  
+  	if ( typeof element.getBoundingClientRect !== undefined ) {
```

Comment:
I don't think it's necessary anymore; we even dropped it in jQuery and Angular supports fewer browsers.